pkg/secrets/filedriver/filedriver.go

```diff
@@ -33,6 +33,11 @@ type Driver struct {
 func NewDriver(rootPath string) (*Driver, error) {
 	fileDriver := new(Driver)
 	fileDriver.secretsDataFilePath = filepath.Join(rootPath, secretsDataFile)
+	// the lockfile functions requre that the rootPath dir is executable
+	err := os.MkdirAll(rootPath, 0700)
+	if err != nil {
```

Comment:
Both lines could be combined, but I'm not blocking on that.

Reply:
Updated as well :)